handbook/engineering/releases/patch_release_issue_template.md

```diff
@@ -10,73 +10,88 @@ Arguments:
-    git checkout '$MAJOR.$MINOR'
-    git tag -a 'v$MAJOR.$MINOR.$PATCH' -m 'v$MAJOR.$MINOR.$PATCH' && git push origin 'v$MAJOR.$MINOR.$PATCH'
+    yarn run release release-candidate:create 1
     ```
-- [ ] Wait for the final Docker images to be available at https://hub.docker.com/r/sourcegraph/server/tags.
-- [ ] Run the old and new images at least three times to make sure it starts:
+- [ ] Wait for the release candidate Docker images to be available in [Docker Hub](https://hub.docker.com/r/sourcegraph/server/tags).
+- [ ] Ensure the release candidate starts for upgrades and new instances:
     ```
     # 1. Answer YES to delete /tmp/sourcegraph with the old image
     IMAGE=sourcegraph/server:$MAJOR.$MINOR ./dev/run-server-image.sh
     
     # 2. Answer NO to delete /tmp/sourcegraph with the new image
-    IMAGE=sourcegraph/server:$MAJOR.$MINOR.$PATCH ./dev/run-server-image.sh
+    IMAGE=sourcegraph/server:$MAJOR.$MINOR.$PATCH-rc.1 ./dev/run-server-image.sh
     
     # 3. Answer YES to delete /tmp/sourcegraph with the new image
-    IMAGE=sourcegraph/server:$MAJOR.$MINOR.$PATCH ./dev/run-server-image.sh
+    IMAGE=sourcegraph/server:$MAJOR.$MINOR.$PATCH-rc.1 ./dev/run-server-image.sh
     ```
 
-## Release Kubernetes deployment
-
-In [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph):
+## Stage release
 
-- [ ] Cherry-pick relevant `deploy-sourcegraph` configuration changes from `master` onto the relevant release branch.
-    ```
-    git checkout $MAJOR.$MINOR
-    git pull
-    git cherry-pick <commit0> <commit1> ... # all relevant commits from the master branch
-    git push $MAJOR.$MINOR
-    ```
-- [ ] Trigger an upgrade using the [Update tags workflow](https://github.com/sourcegraph/deploy-sourcegraph/actions?query=workflow%3A%22Update+tags%22) - set `version` to a semver contraint that will apply the version you want (e.g. `~$MAJOR.$MINOR`) and set the branch to the branch you want to upgrade (i.e. `$MAJOR.$MINOR`). Merge the pull request that gets created ([example](https://github.com/sourcegraph/deploy-sourcegraph/pull/863)).
-- [ ] Push the release tag on the commit:
+- [ ] Tag the final release:
     ```
-    git tag v$MAJOR.$MINOR.$PATCH
-    git push origin v$MAJOR.$MINOR.$PATCH
+    yarn run release release-candidate:create final
     ```
+- [ ] Wait for the release Docker images to be available in [Docker Hub](https://hub.docker.com/r/sourcegraph/server/tags).
+- [ ] Open PRs that publish the new release and address any action items required to finalize draft PRs (track PR status via the [generated release campaign](https://k8s.sgdev.org/organizations/sourcegraph/campaigns)):
+  ```sh
+  yarn run release release:stage
+  ```
 
-## Release Docker Compose
-
-- [ ] Release Docker Compose by following [these instructions](https://github.com/sourcegraph/deploy-sourcegraph-docker/blob/master/RELEASING.md)
+## Release
 
-## Update the docs
+<!-- Keep in sync with release_issue_template's "Release" section -->
```

Comment:
My only concern with updating the docs here is that they might get out of sync again. Would linking to the place in the code where the issue templates are created by a better idea?

Reply:
Good idea! I left some comments in the code: [`491de67` (#15548)](https://github.com/sourcegraph/sourcegraph/pull/15548/commits/491de674e481090ad46990935da8cb3708a6840e)